### PR TITLE
[auto-techsupport] Stabilize auto-techsupport tests

### PR DESCRIPTION
**Pull request by ppikh**
[auto-techsupport] Stabilize auto-techsupport tests

Improved tests stability
Old logic expected that specific timestamp will be in last log file. Sometimes last log file could be not the same as we expected(near it it but different file with different old timestamp).
Now instead of checking that oldest timestamp == expected oldest timestamp - we do check that oldest timestamp in [list of expected(possible) oldest timestamps]

Signed-off-by: Petro Pikh <petrop@nvidia.com>

### Description of PR
Some tests were not stable and time to time fail. This commit should increase tests stability.

Summary: Stabilize auto-techsupport tests
Fixes # (issue)

### Type of change
- [ ] Bug fix
- [ ] Testbed and Framework(new/improvement)
- [x] Test case(new/improvement)

### Back port request
- [ ] 201911
- [ ] 202012

#### What is the motivation for this PR?
Stabilize tests, reduce test failures

#### How did you do it?
Changed logic from: oldest timestamp == expected oldest timestamp to oldest timestamp in [ expected oldest timestamps ]

#### How did you verify/test it?
Executed tests

### CODE REVIEW

**Comment by liat-grozovik** [comment]
@roysr-nv could you please help to review?

**Review by roysr-nv — APPROVED** [review]

**Comment by liat-grozovik** [comment]
@ppikh could you please help to extend the PR description of what were the instability parts observed and how you manage to handle/reduce them?

File: tests/show_techsupport/test_auto_techsupport.py [hunk 1]
```diff
@@ -836,40 +840,34 @@ def validate_techsupport_generation(duthost, is_techsupport_expected, expected_c
...
-    new_techsupport_files_num = len(new_techsupport_files_list)
-    logger.info('New files created: {} expected: {}'.format(new_techsupport_files_num,
-                                                            expected_techsupport_files_num))
-    assert new_techsupport_files_num == expected_techsupport_files_num, \
-        'Number of new techsupport files {} does not match number of expected {} files'.format(
-            new_techsupport_files_num,
-            expected_techsupport_files_num)
+    if expected_techsupport_files:
+        # techsupport file creation may took some time after generate dump process already finished
+        assert wait_until(300, 10, 0, is_new_techsupport_file_generated, duthost, available_tech_support_files), \
```
Comment (liat-grozovik) [inline tests/show_techsupport/test_auto_techsupport.py:865]: will it wait the max time or it will poll? want to reduce the test time if possible

Comment (ppikh) [reply tests/show_techsupport/test_auto_techsupport.py:865]: it will poll every 10 sec, up to 300 sec

**Review by liat-grozovik — APPROVED** [review]

**Pull request merged**